### ISSUE TITLE
integration test failure - make test_winrm fails in test_win_fetch

### ISSUE THREAD

**Issue by jhawkesworth** [ansible/ansible#12056]
##### Ansible Version:  ansible 2.0.0 (devel 9bb95b5235) last updated 2015/08/21 06:47:52 (GMT +100)

  lib/ansible/modules/core: (detached HEAD 4721d6d8b5) last updated 2015/08/21 06:49:37 (GMT +100)
  lib/ansible/modules/extras: (detached HEAD 785cd24a02) last updated 2015/08/21 06:49:17 (GMT +100)
  v1/ansible/modules/core: (detached HEAD f8d8af17cd) last updated 2015/06/10 18:29:52 (GMT +100)
  v1/ansible/modules/extras: (detached HEAD 495ad450e5) last updated 2015/06/10 18:30:43 (GMT +100)
  config file = 
  configured module search path = None
##### Ansible Configuration:

Stock ansible configuration
##### Environment:

Ansible controller: Lubuntu 12.04 i686
Managed host (1 \* Windows Server 2012 R2).
##### Summary:

The windows integration tests fail at the first task of test_win_fetch.
##### Steps To Reproduce:

git checkout devel
git fetch upstream
git rebase upstream/devel
git submodule update --init --recursive
ansible --version
cd test/integration
export TEST_FLAGS='-i /etc/ansible/WINDOWS -t test_win_fetch -vvv)'
make test_winrm
##### Expected Results:

All winrm integration tests complete.
##### Actual Results:

Playbook run fails with the following:

```
fatal: [win-testbox1]: FAILED! => {"failed": true, "msg": "ERROR! The module file was not found in configured module paths"}
```

**Comment by jhawkesworth** [comment]
The task it is failing on is a local_action:

```
- name: clean out the test directory
  local_action: file name={{ output_dir|mandatory }} state=absent
  run_once: true
```

Its possible then it is related to #12053 although running with -vvvvv doesn't show any connection information before failure.

For info, ansible-doc -l does report the 'file' module as being present, so it is not simply that module paths have not been set in this case.

**Comment by cchurch** [comment]
@jhawkesworth: #12053 addresses the overall issue -- local_action is still trying to use the winrm connection instead of a local connection.  Since we only search for modules with a .ps1 extension when using a winrm connection, it is unable to find a file.ps1 module.

**Comment by cchurch** [comment]
I've confirmed that the `local_action` fix in 49ca0eb also fixes this issue.

**Comment by jhawkesworth** [comment]
Thanks for this. For some reason I've stopped receiving github notifications so only just noticed this.
